console/src/main/java/org/eclipse/rdf4j/console/util/ConsoleRDFWriter.java

```diff
@@ -0,0 +1,83 @@
+/*******************************************************************************
+ * Copyright (c) 2018 Eclipse RDF4J contributors.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Distribution License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ *******************************************************************************/
+package org.eclipse.rdf4j.console.util;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.eclipse.rdf4j.console.ConsoleIO;
+import org.eclipse.rdf4j.console.Util;
+import org.eclipse.rdf4j.model.Statement;
+
+import org.eclipse.rdf4j.query.QueryResultHandlerException;
+import org.eclipse.rdf4j.rio.RDFFormat;
+import org.eclipse.rdf4j.rio.RDFHandlerException;
+import org.eclipse.rdf4j.rio.helpers.AbstractRDFWriter;
+
+
+/**
+ * Write query results to console
+ * 
+ * @author Bart Hanssens
+ */
+public class ConsoleRDFWriter extends AbstractRDFWriter {
+	private final ConsoleIO consoleIO;
+	private final int consoleWidth;
+	private final Map<String,String> namespaces = new HashMap<>();
+	private int columnWidth;
+	private String separatorLine = "";
+	private String header = "";
```

Comment:
Are these two (above) variables in use?

Reply:
yes, the header is created in the inherited `startQueryResult` method, but not displayed yet - that happens in the inherited `startHeader` method (which does not take parameters)

The separator line is (also) used at the end of the results (a matter of taste actually, removing the line at the end - and at the start - would save two lines of output and a variable; but in previous versions of RDF4J console these lines are present)